ironfish/src/blockchain/database/evmReceiptValue.ts

```diff
@@ -0,0 +1,141 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import type { IDatabaseEncoding } from '../../storage/database/types'
+import { Log } from '@ethereumjs/evm'
+import { RunTxResult } from '@ethereumjs/vm'
+import bufio from 'bufio'
+
+export interface EvmReceiptValue {
+  type: number
+  status: number
+  contractAddress?: Buffer
+  logsBloom: Buffer
+  logs: Log[]
+  gasUsed: bigint
+  cumulativeGasUsed: bigint
+
+  // fields below are included in Geth serialization, but we may not need them
+
```

Comment:
Can we derive `effectiveGasPrice` from somewhere else? Seems like it is worth storing even if it will always be 0 for now.

Reply:
There's a derivation in the ethereumjs implementation of `eth_getTransactionReceipt`: 
```
    const effectiveGasPrice = tx.supports(Capability.EIP1559FeeMarket)
      ? (tx as FeeMarketEIP1559Transaction).maxPriorityFeePerGas <
        (tx as FeeMarketEIP1559Transaction).maxFeePerGas - block.header.baseFeePerGas!
        ? (tx as FeeMarketEIP1559Transaction).maxPriorityFeePerGas
        : (tx as FeeMarketEIP1559Transaction).maxFeePerGas -
          block.header.baseFeePerGas! +
          block.header.baseFeePerGas!
      : (tx as LegacyTransaction).gasPrice
 ```
 
 I think for now it should always be `tx.gasPrice` since we're using legacy transactions, but it looks like in general we can derive it from the transaction and block.